iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtensionLogger.java

```diff
@@ -45,7 +42,7 @@ public void logIterableApiError(retrofit2.Call<?> preparedCall,
       Buffer buffer = new Buffer();
       preparedCall.request().body().writeTo(buffer);
       requestBody = buffer.readUtf8();
-    } catch (IOException e) {
+    } catch (Exception e) {
```

Comment:
Changed to Exception since `preparedCall.request().body().writeTo(buffer)` can produce NPE